vignettes/kalman.Rmd

```diff
@@ -87,10 +88,21 @@ volatility_data <- mcstate::particle_filter_data(data, "step", 1)
 head(volatility_data)
 ```
 
-With all these pieces we can create our particle filter object, specifying that we will use $10^3$ particles
+The particle filter can run with more or fewer particles - this will trade-off runtime with the variance in the estimate:
```

Comment:
```suggestion
The particle filter can run with more or fewer particles - this will trade-off runtime with the variance in the estimate, though in a fairly non-linear manner:
```